### ISSUE TITLE
Corrections, suggestions and ideas for Croatian dictionaries

### ISSUE THREAD

**Issue by josephwright** [josephwright/beamer#267]
Originally reported by: **Ivan Kokan (Bitbucket: [ivankokan](https://bitbucket.org/ivankokan), GitHub: [ivankokan](https://github.com/ivankokan))**

----------------------------------------

Hi,

I've checked all latest beamer Croatian dictionaries and I have some ideas for you how to improve them.

According to new Orthography released by Institute of Croatian Language and Linguistics, some spellings are prioritized/recommended while others are just allowed. This is supposed to be an "official" Croatian orthography (required by law), so I think these corrections must be done:

```
#!latex

translator-environment-dictionary-Croatian.dict
-----------------------------------------------
\providetranslation{disadvantages}{nedostatci}
\providetranslation{Disadvantages}{Nedostatci} % notice there's a bug in this line (line 11) - uppercase "D" should be used!

translator-theorem-dictionary-Croatian.dict
-------------------------------------------
\providetranslation{Problems}{Zadatci}
\providetranslation{problems}{zadatci}

```
Check: http://pravopis.hr/rjecnik/
	=> nedostatak mn. N nedostatci <nedostaci>
	=> zadatak mn. N zadatci <zadaci>

Also, I think there should be lowercase "I" in these three lines, and abbreviation "izvj." should be used, e.g. see Croatian definitions for babelbib:

```
#!latex

translator-bibliography-dictionary-Croatian.dict
------------------------------------------------
\providetranslation{Tech. Rep.}{Tehn.\ izvj.}
\providetranslation{Technical Reports}{Tehni\v cki izvje\v staji}
\providetranslation{Technical Report}{Tehni\v cki izvje\v staj}

```

Number dictionary contains two errors - it should be "eighth" and "twelfth" (maybe you should check other dictionaries about this):

```
#!latex

translator-numbers-dictionary-Croatian.dict
-------------------------------------------
\providetranslation{eighth}{osmi}
\providetranslation{twelfth}{dvanaesti}

```

Here are some corrections and suggestions for basic dictionary:

```
#!latex

translator-basic-dictionary-Croatian.dict
-----------------------------------------
\providetranslation{cont}{nast.}
\providetranslation{See also}{Tako\dj er vidjeti}
\providetranslation{see also}{tako\dj er vidjeti}
\providetranslation{Related Work}{Povezani radovi}
\providetranslation{Index}{Kazalo}
\providetranslation{Sketch of Proofs}{Skica dokaza}
\providetranslation{Sketch of Proof}{Skica dokaza}

```
In which context keyword "To" is used? Maybe "Do" and "do" should be used instead of "Za" and "za"?

"Outline" could be "Pregled", "Shema", "Nacrt" or "Obris"?

I'd rather pick abbreviation ("pril.") xor singular xor plural here:

```
#!latex

\providetranslation{encl}{Prilog/zi}
```

Finally, I think you should consider these translations (I think they're more appropriate):

```
#!latex

translator-theorem-dictionary-Croatian.dict
-------------------------------------------
\providetranslation{Main Theorems}{Glavni teoremi}
\providetranslation{main theorems}{glavni teoremi}
\providetranslation{Main Theorem}{Glavni teorem}
\providetranslation{main theorem}{glavni teorem}
\providetranslation{Key Lemmas}{Klju\v cne leme} % or "Bitne"
\providetranslation{key lemmas}{klju\v cne leme} % or "bitne"
\providetranslation{Key Lemma}{Klju\v cna lema} % or "Bitna"
\providetranslation{key lemma}{klju\v cna lema} % or "bitna"
\providetranslation{Key Observations}{Klju\v cne primjedbe} % or "Bitne", maybe "Zapa\v zanja"
\providetranslation{key observations}{klju\v cne primjedbe} % or "bitne", maybe "zapa\v zanja"
\providetranslation{Key Observation}{Klju\v cna primjedba} % or "Bitna", maybe "Zapa\v zanje"
\providetranslation{key observation}{klju\v cna primjedba} % or "bitna", maybe "zapa\v zanje"
\providetranslation{Observations}{Primjedbe} % maybe "Zapa\v zanja"
\providetranslation{observations}{primjedbe} % maybe "zapa\v zanja"
\providetranslation{Observation}{Primjedba} % maybe "Zapa\v zanje"
\providetranslation{observation}{primjedba} % maybe "zapa\v zanje"

```

I hope this will give us much better Croatian translations.

Kind regards, Ivan Kokan

----------------------------------------
- Bitbucket: https://bitbucket.org/rivanvx/beamer/issue/267

**Comment by josephwright** [comment]
*Original comment by* **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**:

----------------------------------------

Could you attach (or Mercurial pull request) complete versions of the files?

**Comment by josephwright** [comment]
*Original comment by* **Ivan Kokan (Bitbucket: [ivankokan](https://bitbucket.org/ivankokan), GitHub: [ivankokan](https://github.com/ivankokan))**:

----------------------------------------

Hi Joseph! No problem, I'll prepare it today or tomorrow. Should I create my own branch and work inside it? I think Vedran is the most appropriate person to check my proposal.

Kind regards, Ivan

**Comment by josephwright** [comment]
*Original comment by* **Vedran Miletić (Bitbucket: [rivanvx](https://bitbucket.org/rivanvx), GitHub: Unknown)**:

----------------------------------------

Sure, if you just provide a diff it will be fine as well.

**Comment by josephwright** [comment]
*Original comment by* **Ivan Kokan (Bitbucket: [ivankokan](https://bitbucket.org/ivankokan), GitHub: [ivankokan](https://github.com/ivankokan))**:

----------------------------------------

Hi, I'm ready to commit changes or copy patch contents (I can't generate patch file before commiting, but only export its contents in plain format). Which option you prefer?

Kind regards, Ivan

**Comment by josephwright** [comment]
*Original comment by* **Vedran Miletić (Bitbucket: [rivanvx](https://bitbucket.org/rivanvx), GitHub: Unknown)**:

----------------------------------------

Pull request is better. No need for separate branch.

**Comment by josephwright** [comment]
*Original comment by* **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**:

----------------------------------------

@rivanvx Still about I see, at least for Croatian questions :-)

**Comment by josephwright** [comment]
*Original comment by* **Ivan Kokan (Bitbucket: [ivankokan](https://bitbucket.org/ivankokan), GitHub: [ivankokan](https://github.com/ivankokan))**:

----------------------------------------

There's a pull request to be analyzed.

**Comment by josephwright** [comment]
*Original comment by* **Vedran Miletić (Bitbucket: [rivanvx](https://bitbucket.org/rivanvx), GitHub: Unknown)**:

----------------------------------------

Merged.

**Comment by josephwright** [comment]
*Original comment by* **Vedran Miletić (Bitbucket: [rivanvx](https://bitbucket.org/rivanvx), GitHub: Unknown)**:

----------------------------------------

Fixed in 128121e improvements for Croatian translations.

**Comment by josephwright** [comment]
*Original comment by* **Ivan Kokan (Bitbucket: [ivankokan](https://bitbucket.org/ivankokan), GitHub: [ivankokan](https://github.com/ivankokan))**:

----------------------------------------

Great!
